gradle/scripts/ember.gradle

```diff
@@ -2,10 +2,10 @@ apply plugin: "com.moowork.node"
 
 node {
   // Version of node to use.
-  version = '7.8.0'
+  version = '7.10.1'
```

Comment:
Seems some people are experiencing issues building with this node version? Are you able to build successfully? I'll try to investigate this in a bit. See #652 @shpark76 also raised a similar issue